### PR TITLE
Feature/create state of the dapps group and default groups [ui-refresh] (#599)

### LINKED ISSUE #599: Create "State of the DApps" group and integrate JSON
State of the DApps will provide a curated list of Ropsten compatible DApps that will be a group within Status contact list named "State of the DApps".

This depends on EtherCasts/state-of-the-dapps#22

### PR DESCRIPTION
**Pull request by flexsurfer**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)
fixes #599

### Summary:
[comment]: # (Summarise the problem and how the pull request solves it)
Implemented default contact groups json included two groups "Dapps" and "State of the Dapps"

### Steps to test:
- Open Status
- Open Contacts

[comment]: # (PRs will only be accepted if squashed into single commit.)
status: ready

### CODE REVIEW

**Comment by flexsurfer** [comment]
looks like that we need to find a way to add new contacts to the default groups, before we can merge this PR

File: src/status_im/data_store/realm/contact_groups.cljs [hunk 1]
```diff
@@ -42,7 +42,10 @@
 (defn- save-contacts
   [identities contacts]
   (doseq [contact-identity identities]
-    (.push contacts (clj->js {:identity contact-identity}))))
+    (if-let [contact (.find contacts (fn [object _ _]
```
Comment (rasom) [inline src/status_im/data_store/realm/contact_groups.cljs]: `when-not`?

Comment (flexsurfer) [inline src/status_im/data_store/realm/contact_groups.cljs]: I thought for a some time how to do it :) but, 
```
(let [contact]
  (when-not (contact) 
```
It seemed to me not clear, i tried to find `when-not-let` and `if-let` with `nil` seemed fine  :)

Comment (rasom) [inline src/status_im/data_store/realm/contact_groups.cljs]:
```clojure
(when-not (.find contacts ..)
   (.push contacts (clj->js {:identity contact-identity}))
```
will work here, no? Why do we need `let` here?

Comment (flexsurfer) [inline src/status_im/data_store/realm/contact_groups.cljs]: should work, thanks

File: src/status_im/data_store/realm/contact_groups.cljs [hunk 2]
```diff
@@ -42,7 +42,9 @@
 (defn- save-contacts
   [identities contacts]
   (doseq [contact-identity identities]
-    (.push contacts (clj->js {:identity contact-identity}))))
+    (when-not (.find contacts (fn [object _ _]
+                                (= contact-identity (aget object "identity"))))
+      (.push contacts (clj->js {:identity contact-identity})))))
```
Comment (flexsurfer) [inline src/status_im/data_store/realm/contact_groups.cljs]: done

**Comment by flexsurfer** [comment]
@rasom done

**Pull request merged**